live-site/lib/data.ts

```diff
@@ -11,15 +11,15 @@ const liveSiteTabInfo: TabInfo[] = [
 const resources: ResourceBoxProps[] = [
   {
     label: 'Project Guide',
-    link: '#'
+    link: 'https://docs.google.com/document/d/1xhgMtE5XmauOgewQ4M9_rqQd4JQUZQ1TgBW2z-pZfa8/edit?usp=sharing'
```

Comment:
I just talked with PL and we're switching this link to https://docs.google.com/document/d/1kob1aZ6VLQ0li-66onzF09WxdI9sjHPI-zHznm353tk/edit

to include the project submission guide! Sorry for the last minute change